README.md

```diff
@@ -1 +1,135 @@
-# dbt-bq-macros
\ No newline at end of file
+# dbt-bq-macros
+
+This repo contains dbt macros that can be used in bigquery. 
+
+**[Macros](#macros)**
+
+- [log_to_scd](#log_to_scd)
+- [log_to_latest](#log_to_latest)
+- [stg_dedup](#stg_dedup)
+
+## Macros
+
+### log_to_scd ([source](macros/log_to_scd.sql))
+
+This macro transform a log table to a slowly changing dimension table of type 2. The undelying table is assumed to contain log events where not all of them contains actual updaes to the data, metadata columns not of interest is therefore excluded from comparing whether the data changed or not. The resulting table will have rows with a valid_from and valid_to column as well as a version column.
+
+### Macro Arguments
+- `table_`: Name of the table or CTE of the log table from the context where this macro is used
+- `id`: Identifier of the entity
+- `event_id`: Unqiue identifier for each row
+- `updated_at`: Timestamp column indicated when this event happened 
+- `excluded_cols`: Columns to exclude while evaluating whether any data has changed.
+
+### Resulting table
+
+- All columns except the excluding columns plus:
```

Comment:
☝️  is answered in the code. Then I wonder why do we want to exclude columns for `scd` specifically?

Reply:
This is maybe a bit confusing and could be made more clear.

Excluding columns is for example updated_at columns and eventId. These are not used when comparing if the new row contains any changes. The updated at and event id will always be different values. Whether or not they should be excluded in the result can be debated but I think it's less confusing since the updated_at field will be replaced by valid_from and valid_to and the event id doesn't make much sense any more.

Reply:
Thank you, makes sense 👍